extensions/amp-analytics/0.1/sandbox-vars-allowlist.js

```diff
@@ -19,34 +19,35 @@
  * @const {!Object<string, boolean>}
  */
 export const SANDBOX_AVAILABLE_VARS = {
-  'RANDOM': true,
-  'CANONICAL_URL': true,
+  'AMPDOC_HOST': true,
+  'AMPDOC_HOSTNAME': true,
+  'AMPDOC_META': true,
```

Comment:
Ditto addition/sort.

I'm not sure when "sandbox" rules apply, so I'm not sure if this should be added or not.

Reply:
Sandbox rule applies when the `<amp-analytics>` component is inserted into the the doc by a 3rd party extension. 
It is used to prevent 3rd service from collecting user information. I don't think any meta data contain user information today, but maybe you want to exclude it just to be safe.

Reply:
Done.